src/bindings/config.ts

```diff
@@ -13,10 +13,13 @@ import {
 } from "../configs";
 import { ajv } from "../utils";
 import { Context } from "probot";
+import { getPrivateKey } from "../utils/private";
 
 export const loadConfig = async (context: Context): Promise<BotConfig> => {
   let configFile: any = {};
   configFile = await context.config("ubiquibot-config.yml");
+  const keyData = await getPrivateKey();
+  const privateKey = keyData !== undefined ? keyData : process.env.UBIQUITY_BOT_EVM_PRIVATE_KEY !== undefined ? process.env.UBIQUITY_BOT_EVM_PRIVATE_KEY : "";
```

Comment:
`  const privateKey = keyData ?? process.env.UBIQUITY_BOT_EVM_PRIVATE_KEY  ?? "";`

we can make it simple as above

Reply:
updated